intranet/apps/eighth/tests.py

```diff
@@ -227,3 +228,52 @@ def test_absence_email(self):
 
         msg = absence_email(es1)
         self.assertIn("Jan. 1, 2015 (A)", msg.body)
+
+    def test_take_attendance_zero(self):
+        """ Make sure all activities with zero students are marked as having attendance taken when button is pressed. """
+        self.make_admin()
+        block1 = self.add_block(date='3000-11-11', block_letter='A')
+
+        room1 = self.add_room(name="room1", capacity=1)
+
+        act1 = self.add_activity(name='Test Activity 1')
+        act1.rooms.add(room1)
+        schact1 = self.schedule_activity(act1.id, block1.id)
+        schact1.attendance_taken = False
+        schact1.save()
+
+        response = self.client.post(reverse('eighth_admin_view_activities_without_attendance') + "?" + urlencode({"block": block1.id}), {"take_attendance_zero": "1"})
```

Comment:
@ezwang pretty sure there's a better way to do this then hard-coding the query string.